src/createHtml.jsx

```diff
@@ -140,7 +140,7 @@ export default function(options) {
           {visualWebsiteOptimizer ? <script type="text/javascript" dangerouslySetInnerHTML={{__html: vwo2()}}></script> : null}
           {visualWebsiteOptimizer ? <script type="text/javascript" dangerouslySetInnerHTML={{__html: vwo3()}}></script> : null}
 
-          {clippyChatTimeout ? <link key="clippy-chat" href="/shared/content/dist/clippy-chat.css" rel="stylesheet"/> : null}
+          {clippyChatTimeout ? <link key="clippy-chat" href="/shared/content/dist/clippy-chat.css?v=1.0.0.362" rel="stylesheet"/> : null}
```

Comment:
Agreed, and it was not included at first, added after pairing.. In reality, the query string does nothing... It should be removed.